src/utils/method-helpers.js

```diff
@@ -28,3 +29,6 @@ const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
 
 export const delay = (delayed, ms) =>
   Promise.all([delayed, sleep(ms)]).then(([data]) => data);
+
+export const translate = (key, lang, interpolation = null) =>
```

Comment:
Yes thanks for bringing this up. So there were two things I considered:

1. Find the language locale from the user's device and use it directly. Don't allow the user to change the language in the app and if they want to they would have to change it on their device.
2. Allow the user to change the language on their device.

The pros of 1) is that we wouldn't have to pass in `lang` the way I do now in every copy in the entire app. Just calling the translate function should suffice _I think_.

The issue with this however is that the user may not even know different languages if their device language isn't supported. I kinda think it's nicer to allow the user to change the language on the app directly.

So to allow for this, I added a `lang` parameter to `authReducer`. Throughout the entire app, I just pass this parameter down to this method and when the language is changed in the `userProfileScreen` - this gets reflected in all the copy in the app immediately.

What do you think, open to all kinds of suggestions and advice. Thanks for the help reviewing this PR mate :raised_hands: